server/src/main/java/org/candlepin/auth/permissions/AsyncJobStatusPermission.java

```diff
@@ -59,6 +62,10 @@ public Criterion getCriteriaRestrictions(Class entityClass) {
 
     @Override
     public boolean canAccessTarget(AsyncJobStatus target, SubResource subResource, Access required) {
+        if (isOwnerInvalid(target)) {
```

Comment:
We are (slowly) moving toward not doing this type of check in the permissions and, instead, checking before the job is created. Once the job is created, it's being assumed to have cleared all of these sorts of checks.

If this is to facilitate some kind of targeting for cancelling/fetching status, we should look at more robust solutions, and/or checks to do in the appropriate resource(s).

Reply:
You are right. I added the check for cancelling/fetching as according to the spec tests it was expected behavior. I created methods get(jobId) and cancel(jobId) in JobManager and moved the checks there.

Reply:
For now let's just drop the check entirely. This is something that needs to be fleshed out more, and I really don't want jobs to be cancellable or such based on the arguments or metadata.

Reply:
Ok. I dropped the check and added skip to its spec test.